### PR TITLE
Added debugging tool to tests

### PR DESCRIPTION
**Pull request by hAdamm**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
         "mockery/mockery": "0.9.7",
         "nette/tester": "~1.3",
         "nette/caching": "~2.2",
-        "squizlabs/php_codesniffer": "~2.5"
+        "squizlabs/php_codesniffer": "~2.5",
+        "tracy/tracy": "~2"
```
Comment (ZZromanZZ) [inline composer.json]: ~2.4

File: .gitignore [hunk 2]
```diff
@@ -25,4 +25,5 @@ vendor/
 code-checker/
 tests/temp/*
 !tests/temp/.placeholder
-composer.lock
\ No newline at end of file
+composer.lock
+output
```
Comment (ZZromanZZ) [inline .gitignore]: what is output for ?

File: tests/bootstrap.php [hunk 3]
```diff
@@ -8,6 +8,26 @@
 Tester\Environment::setup();
 date_default_timezone_set("Europe/Prague");
 
+\Tracy\Debugger::$logDirectory =  __DIR__ . "/log";
+
+$_prevEx = set_exception_handler(function($e) use (& $_prevEx) {
+    if (!$e instanceof Tester\AssertException) Tracy\Debugger::log($e, Tracy\ILogger::EXCEPTION);
```
Comment (ZZromanZZ) [inline tests/bootstrap.php]: backslash before Tracy\ILogger::EXCEPTION ?

File: tests/bootstrap.php [hunk 4]
```diff
@@ -8,6 +8,26 @@
...
+    if ($_prevEx !== null) {
+        $_prevEx($e);
+    }
+});
+
+$_prevEr = set_error_handler(function($severity, $message, $file, $line) use (& $_prevEr) {
+    if (error_reporting() === 0) return false;
+    $e = new \ErrorException($message, 0, $severity, $file, $line);
+    Tracy\Debugger::log($e, Tracy\ILogger::EXCEPTION);
```
Comment (ZZromanZZ) [inline tests/bootstrap.php]: backslash before Tracy\ILogger::EXCEPTION ?

File: .gitignore (hunk 2 above)
Comment (hAdamm) [inline .gitignore]: for nette tester

File: composer.json (hunk 1 above)
Comment (hAdamm) [inline composer.json]: ok

**Pull request merged**